entities/params/segmentparams.go

```diff
@@ -0,0 +1,14 @@
+package params
+
+import (
+	"strings"
+)
+
+// SegmentParams represents request body for POST /api/segments
+type SegmentParams struct {
```

Comment:
We'll need to agree on some naming here, on some params we prefix them with the http method like PostFoo, PutBar etc. and here it's just SegmentParams.

I think it's better if we prefix them with the HTTP method.

Reply:
@FilipNikolovski  Yes this is the same case as before Segment is using same params for put,post

Reply:
@FilipNikolovski In this case i think it's better to have 1 struct